app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -292,10 +294,17 @@ fun AppViewContent(
       )
     }
     if (app.isAppCoins) {
+      val bonusTag = "bonus-banner"
+      val bonusTitle = "Bonus"
```

Comment:
Hardcoded string. Probably should be either translated recourse or fetched from network